backend/compact-connect/lambdas/python/common/cc_common/data_model/data_client.py

```diff
@@ -2905,7 +2905,7 @@ def encumber_home_jurisdiction_license_privileges(
         compact: str,
         provider_id: str,
         jurisdiction: str,
-        adverse_action_id: str,
+        adverse_action_id: UUID,
```

Comment:
You'll also need to update the data events handler, where this gets called, to ensure that actually does pass in a UUID.